### PR TITLE
#161382343 Setup environment for writing tests

### PR DESCRIPTION
**Pull request by emukungu**
#### What does this PR do?
Setup environment for writing tests

#### Description of Task to be completed?
All components should have end to end tests

#### How should this be manually tested?
 - Clone the repository.
 - Install application dependencies using`yarn`
 - run the server using `yarn start:dev`
 - run the tests using `yarn test`

#### What are the relevant pivotal tracker stories?
#161382343

#### Screenshots 
screen shot 2018-11-27 at 2 25 20 pm

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,40 +1,39 @@
-const path = require('path');
-const webpack = require('webpack');
-const HTMLWebpackPlugin = require('html-webpack-plugin');
+const path = require("path");
```
[the remark this answers is not in the document]
Comment (emukungu) [reply webpack.config.js:1]: resolved

File: tests/App.test.jsx [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
+import "babel-polyfill";
+import React from "react";
+import { mount } from "enzyme";
+import puppeteer from "puppeteer";
+
+import App from "../src/App";
+
+describe("<App />", () => {
+	it("should render without crashing", () => {
+		expect(mount.bind(null, <App />)).not.toThrow();
```
[the remark this answers is not in the document]
Comment (emukungu) [reply tests/App.test.jsx:10]: resolved

File: tests/App.test.jsx [hunk 3]
```diff
@@ -1,9 +1,21 @@
-import React from 'react';
-import { mount } from 'enzyme';
-import App from '../src/App';
-
-describe('<App />', () => {
-  it('should render without crashing', () => {
-    expect(mount.bind(null, <App />)).not.toThrow();
-  });
+import "babel-polyfill";
+import React from "react";
```
[the remark this answers is not in the document]
Comment (emukungu) [reply tests/App.test.jsx:2]: resolved

File: tests/__mocks__/fileMock.js [hunk 4]
```diff
@@ -1 +1 @@
-module.exports = '';
+module.exports = "";
```
[the remark this answers is not in the document]
Comment (emukungu) [reply tests/__mocks__/fileMock.js:1]: resolved

File: webpack.config.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (emukungu) [reply webpack.config.js:1]: resolved

File: tests/App.test.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (emukungu) [reply tests/App.test.jsx]: resolved

File: tests/App.test.jsx [hunk 5]
```diff
@@ -1,9 +1,21 @@
...
+import { mount } from "enzyme";
+import puppeteer from "puppeteer";
+
+import App from "../src/App";
+
+describe("<App />", () => {
+	it("should render without crashing", () => {
```
[the remark this answers is not in the document]
Comment (emukungu) [reply tests/App.test.jsx:9]: resolved

File: tests/App.test.jsx [hunk 6]
```diff
@@ -1,9 +1,21 @@
...
-    expect(mount.bind(null, <App />)).not.toThrow();
-  });
+import "babel-polyfill";
+import React from "react";
+import { mount } from "enzyme";
+import puppeteer from "puppeteer";
+
+import App from "../src/App";
+
+describe("<App />", () => {
```
[the remark this answers is not in the document]
Comment (emukungu) [reply tests/App.test.jsx:8]: resolved

File: tests/App.test.jsx [hunk 7]
```diff
@@ -1,9 +1,21 @@
...
-describe('<App />', () => {
-  it('should render without crashing', () => {
-    expect(mount.bind(null, <App />)).not.toThrow();
-  });
+import "babel-polyfill";
+import React from "react";
+import { mount } from "enzyme";
+import puppeteer from "puppeteer";
+
+import App from "../src/App";
```
[the remark this answers is not in the document]
Comment (emukungu) [reply tests/App.test.jsx:6]: resolved

**Review by cdvx — APPROVED** [review]
Looks good to me 👍🏾

**Review by johnkegz — APPROVED** [review]
All checks are passing. well done

**Pull request merged**